packages/common/src/api/tan-query/utils/isValidId.ts

```diff
@@ -0,0 +1,3 @@
+export const isValidId = (id: number | null | undefined | string) => {
```

Comment:
It finally happened.
And I'm resisting a conversation over the best way to check if something is a number :-)